node.gypi

```diff
@@ -194,7 +194,18 @@
             },
           },
           'conditions': [
-            ['OS!="aix" and node_shared=="false"', {
+            ['OS=="linux" and node_shared=="false" \
+              and target_arch=="x64" \
+              and node_use_large_pages=="true"', {
+              'ldflags': [
+                 '-Wl,-T <(PRODUCT_DIR)/../../ld.implicit.script',
```

Comment:
This should be:
```
                '-Wl,-T src/ld.implicit.script',
```
Using `src/` after you move `ld.implicit.script` to `src`

Reply:
@refack wouldn't this be <(PRODUCT_DIR)/../../src/large_pages/ld.implicit.script?